pippo-session-parent/pippo-session-jdbc/src/main/java/ro/pippo/session/jdbc/JDBCSessionDataStorage.java

```diff
@@ -24,10 +24,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import ro.pippo.core.PippoRuntimeException;
-import ro.pippo.session.SerializationSessionDataTranscoder;
-import ro.pippo.session.SessionData;
-import ro.pippo.session.SessionDataStorage;
-import ro.pippo.session.SessionDataTranscoder;
+import ro.pippo.session.*;
```

Comment:
I prefer to not use wildcard import (no `*`)

Reply:
They are some files in in this PR that use wildcard import

Reply:
Oh damn, me too, the IDE must have "optimized" that for me